### PR TITLE
fix: update login to remove old flow in some places

### PR DESCRIPTION
**Pull request by nickreynolds**
- better messaging when no tasks/challenges

### CODE REVIEW

File: packages/components/src/TCRUserDashboard/DashboardTextComponents.tsx [hunk 1]
```diff
@@ -38,6 +38,24 @@ export const ChallengesWonLabelText: React.FunctionComponent = props => <>Challe
...
+
+export const NoChallenges: React.FunctionComponent = props => (
+  <StyledTransferTokenTitle>
+    <h3>No Challenges</h3>
+    <p>
+      You haven't participated in any challenges yet.
+    </p>
+  </StyledTransferTokenTitle>
+);
+
```
Comment (tobek) [inline packages/components/src/TCRUserDashboard/DashboardTextComponents.tsx]: Excellent additions.

File: packages/dapp/src/components/Dashboard/DashboardActivity.tsx [hunk 2]
```diff
@@ -307,9 +315,13 @@ class DashboardActivity extends React.Component<
           const refetchUserChallengeData = (): void => {
             refetch();
           };
-
           if (error) {
-            return <ErrorLoadingDataMsg />;
+            console.log("error.ToString(): ", error.toString())
```
Comment (tobek) [inline packages/dapp/src/components/Dashboard/DashboardActivity.tsx]: :boom:

File: packages/newsroom-signup/src/AuthWrapper.tsx [hunk 3]
```diff
@@ -90,104 +58,21 @@ class AuthWrapperComponent extends React.Component<RouteComponentProps<AuthParam
...
-  private renderSignupLogin(isNewUser: boolean): JSX.Element {
+  private renderSignupLogin(): JSX.Element {
     return (
       <Wrapper>
         <SignupLoginInnerWrap>
           <OBSectionTitle>Add your Newsroom to Civil</OBSectionTitle>
           <OBSectionDescription>
-            Create an account to add your newsroom to Civil. First, please enter your email address. Your email is used
-            to send account-related updates from Civil.
+            Sign Up or Login to add your Newsroom to Civil.
```
Comment (tobek) [inline packages/newsroom-signup/src/AuthWrapper.tsx]: Minor, should be "Sign Up or Log In" (or maybe "Sign up or log in")

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.22 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**